src/components/interaction-designer/ResourceViewer.vue

```diff
@@ -23,10 +24,16 @@ export const DEBOUNCE_FLOW_PERSIST_MS = 1500
 export default {
   name: 'ResourceViewer',
   mixins: [Lang],
+  props: {
+    visibleBlocks: {
+      type: Array,
+      default: () => null,
+    },
```

Comment:
Hiding blocks is necessary for keeping the state of resource accordions.

Reply:
Could you elaborate a bit - why a list of UUIDs instead of actual blocks?
(I agree that it is better but I can't articulate)

Reply:
UUIDs are used to avoid possible confusion, as this is a list of block IDs not to display.

The blocks now hidden `v-show` instead of conditional `v-if` rendering. This allows persisting the state of accordions, which are nested components – otherwise they would be recreated and the information about collapsed/expanded lost.